tox.ini

```diff
@@ -1,17 +1,36 @@
 [tox]
-envlist = py27
+envlist = py{35,36,37}
 
-[testenv]
-usedevelop = true
+[base]
 deps =
-    -rrequirements.txt
     pyftpdlib
     parameterized
     pytest-xdist
     pytest-cov
-    miniboa
+    miniboa >= 1.0.8
     biplist
+
+[testenv]
+usedevelop = true
+deps =
+    -rrequirements.txt
+    {[base]deps}
 commands =
     py.test {posargs:--tb=short --cov=linkcheck tests}
```

Comment:
IIRC new versions of pytest drop the `py.test` executable and want you to use `pytest`.

(Are we pinning an old version of pytest in requirements.txt, or have I misremembered and the old py.test command alias still works in recent versions?)

Reply:
No pinning here. `py.test` still works with pytest 4.5.0. Tempting to just change it, but it seems like something extra.

Reply:
I probably misread the "Removed all py.test-X* entry points" note in pytest 3.0.0's changelog and ended up with the wrong impression.